RubeesEat.UnitTests/Model/TestDomain.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections.Immutable;
+using RubeesEat.Model;
+
+namespace RubeesEat.UnitTests.Model;
+
+public static class TestDomain
+{
+    public static readonly List<Person> TestPersons =
+    [
+        Person.Create("DefaultUserFirstName", "DefaultUserLastName"), 
+        Person.Create("SecondUserFirstName", "SecondUserLastName"),
+        Person.Create("ThirdUserFirstName", "ThirdUserLastName"),
```

Comment:
For asserts it might be better to have direct access to the persons. I would also give them actual personas to make the tests more readable. e.g. `TestDomain.Alice`, `TestDomain.Bob` etc. (instead of `TestDomain.TestPersons[0]`). The list of persons would then use these properties to fill the list. e.g. `TestDomain.AllPeople`, which contains all people instances.